### ISSUE TITLE
Package deploy page have error

### ISSUE THREAD

**Issue by ddurieux** [fusioninventory/fusioninventory-for-glpi#891]
---

Author Name: **David Durieux** (@ddurieux)
Original Redmine Issue: 1692, http://forge.fusioninventory.org/issues/1692
Original Date: 2012-06-08
Original Assignee: David Durieux

---

```
Parse error: syntax error, unexpected '[' in /Donnees/www/data-dist/glpi080/plugins/fusinvdeploy/inc/package.class.php on line 335 Call Stack: 0.0001 635936 1. {main}() /Donnees/www/data-dist/glpi080/plugins/fusinvdeploy/front/package.php:0 0.0513 6167912 2. __autoload() /Donnees/www/data-dist/glpi080/plugins/fusinvdeploy/front/package.php:57 

```

**Comment by ddurieux** [comment]
---

Original Redmine Comment
Author Name: **David Durieux** (@ddurieux)
Original Date: 2012-06-08T08:50:27Z

---

Fixé : commit:6cbb7c1d

**Comment by ddurieux** [comment]
---

Original Redmine Comment
Author Name: **Gonéri Le Bouder** (Gonéri Le Bouder)
Original Date: 2012-07-03T18:29:45Z

---

The bug is open for weeks now :( Please David, release a fusioninventory-for-glpi with the fix.
